rust-toolchain

```diff
@@ -1 +1 @@
-nightly-2019-06-14
+nightly
```

Comment:
I had this fixed BC the latest nightly usually doesn't have all the components we are using..
https://rust-lang.github.io/rustup-components-history

I think `clippy` /  `fmt` is going to fail on nightly eventually if not currently failing

Reply:
Ok, I've pinned it at the latest version with rustfmt which at the moment is 2020-07-27